ppl/zqd/handlers_test.go

```diff
@@ -412,17 +406,17 @@ func TestRequestID(t *testing.T) {
 	})
 }
 
-func TestPostZngLogs(t *testing.T) {
+func TestPostZsonLogs(t *testing.T) {
```

Comment:
I guess technically this should have been `TestPostTzngLogs()` before, but it was probably a holdover from the days that ZNG was always the text-based format. In any case, the name seemed even more wrong now that it's all ZSON, so I went ahead and changed it.